pkg/tracker/balance_test.go

```diff
@@ -22,9 +25,8 @@ func TestStringId(t *testing.T) {
 	logger := logSetup("debug")
 	tracker := NewBalanceTracker(logger)
 	res := tracker.String()
-	if res != BalanceTrackerName {
-		t.Fatal("didn't return expected string", BalanceTrackerName)
-	}
+
+	testutil.Assert(t, res == BalanceTrackerName, "didn't return expected string", BalanceTrackerName)
```

Comment:
```suggestion
	testutil.Equals(t, res, BalanceTrackerName, "didn't return expected string", BalanceTrackerName)
```

Not a blocker just a personal preference.